app/src/main/java/com/example/teumteum/ui/myhome/MySleepPatternSettingFragment.kt

```
@@ -113,6 +113,26 @@ class MySleepPatternSettingFragment : Fragment() {
+
+        if (initialTime != null) {
+            val isPm = initialTime.hour >= 12
+            ampmPicker.value = if (isPm) 1 else 0
+
+            val hour12 = when (val h = initialTime.hour % 12) {
+                0 -> 12
+                else -> h
+            }
+            hourPicker.value = hour12
+
+            val minuteStr = String.format("%02d", initialTime.minute)
```

detekt (1.23.8)

[warning] String.format("%02d", initialTime.minute) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)